text/0002-split-coinstake-transaction/0002-split-coinstake-transaction.md

```diff
@@ -37,21 +35,25 @@ Only the coinbase `vin[0]` has been used to carry meta-data (e.g. signaling P2SH
 Splitting the coinstake transaction into a *monetary creation* and a *coin-age consumption* transaction, has the side effect of creating small value outputs.
 Because those outputs are very unlikely to mint new blocks, node implementations are adviced to join them with a future *coin-age consumption* transaction.
 
-The minter is awarded a budget of 1 KB for the *coin-age consumption* transaction in order to adjust their UTXO table as they see fit.
-If the *coin-age consumption* transaction is greater than 1 KB, it will have to include the standard fee less 1 KB.
+The minter is currently awarded a budget of 1 KB for the coinstake transaction, and will continue to have this power.
+While the coinbase transaction does not currently have this limitation, the *monetary creation* transaction will also be limited to 1 KB.
```

Comment:
I do not agree with a 1KB budget for the coinbase. It is not needed as the coinbase is not allowed to have any inputs.

Reply:
Is there anything that prevents the minter from turning a 1 PPC block reward into 100 UTXOs?  What is the harm in additional spam protection?  What is your response to this thread:
https://talk.peercoin.net/t/unlimited-coinbase-data/7856

Reply:
I see, we do not have the opportunity cost like bitcoin miners do.
I'd rather implement a block standardness rule that would orphan to large coinbases like is implemented to prevent multi chain minting (this way an operational limit is set rather than a protocol limit).
For peer-to-peer mining pools it can make sense to split the coinbase in many outputs, so I do not like limiting the possibilities here as it would reduce bitcoin compatibility.

Since this issue already exist in current protocol, I believe it should be debated separately.
What is your opinion here @sigmike ?

Reply:
I agree this coinbase issue is unrelated to this RFC and should be discussed elsewhere.

I'd rather use a protocol rule to fix it though. Having custom block validity rules on nodes reduces the network stability. And we can fix the problem with a soft fork.